### ISSUE TITLE
Is there a way to -nolisten tcp ?

### ISSUE THREAD

**Issue by klrvagabond** [pelya/xserver-xsdl#63]
I may be missing something really basic because I'm not much of an Android person, but is there a way to change the arguments used when calling xserver-xsdl? I'd like to be able to disable tcp when it's not necessary.
Incidentally, I'm a huge fan. Being able to run 'real' X on a 7" tablet is VASTLY better than trying to use VNC. Thanks very much for all of the work.

**Comment by pelya** [comment]
_Change device config_ → _Data installation location_ → _Specify directory_
→ do not change directory, press _OK_ → change parameters to
_XSDL -nolisten tcp_

**Comment by klrvagabond** [comment]
Thanks! I'd seen references that should've pointed me over there, but I had misinterpreted 'do not change directory' to 'tap on the one that's already selected' instead of 'specify directory and then don't make changes'.
I really appreciate the quick response.

**Comment by pelya** [comment]
Yeah, I will add a simpler method to change commandline arguments, this one
was intended for debugging only.
